ptvsd/pathutils.py

```diff
@@ -56,8 +56,7 @@ def _get_actual_filename(self, name):
             dir_names = os.listdir(os.getcwd())
             fragments = list(filter(
                             lambda x: x.upper() == path_fragment1, dir_names))
-        for d in dirs[1:]:
-            fragments += ["{}[{}]".format(d[:-1], d[-1])]
```

Comment:
What was the old code actually trying to do with `[]` pattern in the glob? It looks like it should be a no-op...

Reply:
I think that was a copying error. This was there to detect .pyc and .py files. It had nothing to do with selecting folders.